pkg/api/v1alpha1/kepler_types.go

```diff
@@ -180,6 +188,36 @@ type Condition struct {
 
 // KeplerStatus defines the observed state of Kepler
 type KeplerStatus struct {
+	// The number of nodes that are running at least 1 kepler pod and are
+	// supposed to run the kepler pod.
+	CurrentNumberScheduled int32 `json:"currentNumberScheduled"`
+
+	// The number of nodes that are running the kepler pod, but are not supposed
+	// to run the kepler pod.
+	NumberMisscheduled int32 `json:"numberMisscheduled"`
+
+	// The total number of nodes that should be running the kepler
+	// pod (including nodes correctly running the kepler pod).
+	DesiredNumberScheduled int32 `json:"desiredNumberScheduled"`
+
+	// numberReady is the number of nodes that should be running the kepler pod
+	// and have one or more of the kepler pod running with a Ready Condition.
+	NumberReady int32 `json:"numberReady"`
+
+	// The total number of nodes that are running updated kepler pod
+	// +optional
+	UpdatedNumberScheduled int32 `json:"updatedNumberScheduled,omitempty"`
+
+	// The number of nodes that should be running the kepler pod and have one or
+	// more of the kepler pod running and available
+	// +optional
+	NumberAvailable int32 `json:"numberAvailable,omitempty"`
+
+	// The number of nodes that should be running the
+	// kepler pod and have none of the kepler pod running and available
+	// +optional
+	NumberUnavailable int32 `json:"numberUnavailable,omitempty"`
+
```

Comment:
can we somehow reduce these ? some of these may be transient. e.g. should we keep both `numberAvailable` and `numberReady` ?

Reply:
I think we should maintian 1-1 mapping to the daemonset.